tfhe/src/core_crypto/algorithms/test/lwe_encryption.rs

```diff
@@ -1012,6 +1012,8 @@ fn lwe_compact_public_encrypt_prove_verify_decrypt_custom_mod<Scalar>(
     let message_modulus_log = params.message_modulus_log;
     let encoding_with_padding = get_encoding_with_padding(ciphertext_modulus);
 
+    let metadata = [b'c', b'o', b'r', b'e'];
```

Comment:
could also be expressed as `"core".as_bytes()`

Reply:
hmm, I don't know if I want to go with that style given utf-8 is variable length